pages/index.js

```diff
@@ -1,46 +1,90 @@
 import React from "react";
-import Link from "next/link";
-//import fetch from "isomorphic-unfetch";
-//import absoluteUrl from "next-absolute-url";
+import fetch from "isomorphic-unfetch";
 import Layout from "../src/components/Layout";
 import { useFetchUser } from "../src/lib/user";
 
+function getWeekNumber(dt) {
+  dt = new Date(dt);
+  var tdt = new Date(dt.valueOf());
+  var dayn = (dt.getDay() + 6) % 7;
+  tdt.setDate(tdt.getDate() - dayn + 3);
+  var firstThursday = tdt.valueOf();
+  tdt.setMonth(0, 1);
+  if (tdt.getDay() !== 4) {
+    tdt.setMonth(0, 1 + ((4 - tdt.getDay() + 7) % 7));
+  }
+  return 1 + Math.ceil((firstThursday - tdt) / 604800000);
+}
+
```

Comment:
you can use [date-fns/getWeek](https://date-fns.org/v2.9.0/docs/getWeek)

Reply:
No longer in use in the next version of the project...